app/helpers/app_form_builder.rb

```diff
@@ -0,0 +1,18 @@
+class AppFormBuilder < ActionView::Helpers::FormBuilder
+
+  def text_field(method, options={})
+    has_errors = @object.errors[method].any?
+
+    @template.content_tag(:div, class: 'govuk-form-group' + (has_errors ? ' govuk-form-group--error' : '')) do
+      #TODO label as option or parameter?
+      @template.label(:label, method, options[:label], class: 'govuk-label') +
+        (options[:hint] ? @template.content_tag(:span, options[:hint], class: 'govuk-hint') : '') +
+      #TODO better wrapping for errors - see design manual. There should be id. Add span with visually-hidden.
```

Comment:
Ano, su tam aj nejake `aria-describedby` - inak niekedy sa pouziva ja fieldset ako heading, cize to mozno cele bude este komplikovanejsie. https://design-system.service.gov.uk/components/fieldset/

Reply:
Pridal som aria-describedby pre hinty a errory